staging/src/k8s.io/dynamic-resource-allocation/structured/allocator_test.go

```diff
@@ -2038,6 +2182,520 @@ func TestAllocator(t *testing.T) {
 				deviceAllocationResult(req0SubReq1, driverA, pool1, device1, false),
 			)},
 		},
+		"partitionable-devices-single-device": {
```

Comment:
one test I don't see is a capacity pool with multiple counters, maybe I missed it? I see multiple capacity pools, but not multiple counters in one pool

Reply:
Added